src/VesselConnectButton.tsx

```diff
@@ -7,14 +7,19 @@ type VesselConnectButtonProps = {
   children?: React.ReactNode;
   style?: React.CSSProperties;
   className?: string;
+  getLinkToken: () => Promise<string>; // Retrieves and returns a linkToken
 } & ClientConfig;
 
 export default function VesselConnectButton({
   children = 'Connect CRM',
   style,
   className,
+  getLinkToken,
   ...config
 }: VesselConnectButtonProps) {
+  if (!getLinkToken) {
+    throw new Error('Prop `getLinkToken` callback must be present.');
+  }
```

Comment:
Will this cause backwards compatbility issues?

Reply:
There's not really an issue of backwards compatibility when you're talking about an NPM package, at least not in the sense that we're talking about here. The package itself is *versioned* and the consumer gets to pick which version they want to use depending on their upgrade timeline (that's what a package-lock file is for).

Now, there *is* an idea of supported versions - but that's an issue of making sure the non-packaged code is backwards compatible. Eventually what we'll say is that our API is compatible with some range of `useVesselLink` versions, but that's 100% a later issue, for now we'll support all versions until our users can upgrade to this version, then we'll just stop supporting non-current versions.

Reply:
But we'll need the server updates to go live before this

Reply:
kk cool works with me since we version our npm packages